src/sip/transp.c

```diff
@@ -334,7 +334,10 @@ static void sip_recv(struct sip *sip, const struct sip_msg *msg,
 
 	if (msg->req) {
 		if (!have_essential_fields(msg)){
-			(void)sip_reply(sip, msg, 400, "Bad Request");
+			if (!pl_strcmp(&msg->met, "ACK"))
+				DEBUG_WARNING("Received bad ACK request\n");
```

Comment:
WARNING is perhaps a bit strict, perhaps INFO or DEBUG is better.

Remember what they said at IETF; Be conservative in what you send, be liberal in what you receive.

Perhaps also add some more info in the log message, for example
where is came from ?

Reply:
OK, I'll try to improve the message. About being liberal, pjsip does accept ACK w/o Max-Forwards.  It really does not cause any harm in ACK.